test/integration/copy_paste_spec.js

```diff
@@ -197,4 +198,116 @@ describe("Copy and paste", () => {
       );
     });
   });
+
+  describe("Copy/paste and ligatures (but without selecting all)", () => {
+    let pages;
+
+    beforeAll(async () => {
+      pages = await loadAndWait(
+        "copy_paste_ligatures.pdf",
+        "#hiddenCopyElement",
+        100,
+        async page => {
+          await page.waitForFunction(
+            () => !!window.PDFViewerApplication.eventBus
+          );
+          await waitForTextLayer(page);
+        }
+      );
+    });
+
+    afterAll(async () => {
+      await closePages(pages);
+    });
+
+    it("must check that the ligatures have been removed when the text has been copied", async () => {
```

Comment:
We already have a test-case in this file with the *same exact* name, which we've seen recently can be annoying when trying to track down intermittent issues; so please change this one slightly.